### PR TITLE
Added a songs tab to the music screen

### PR DESCRIPTION
**Pull request by poisdeux**
The songs tab lists all available songs. <s>This allows the user
to play songs that are not part of an album.</s>

This should provide a fix for issues #160 

AFAICT it doesn't yet provide a fix for issue #7.

### CODE REVIEW

File: app/src/main/java/org/xbmc/kore/provider/MediaProvider.java [hunk 1]
```diff
@@ -80,6 +80,7 @@
     private static final int SONGS_ARTIST = 801;
     private static final int SONGS_ALBUM = 802;
     private static final int SONGS_ID = 803;
+    private static final int SONGS_ALL_HOST = 804;
```
Comment (SyncedSynapse) [inline app/src/main/java/org/xbmc/kore/provider/MediaProvider.java]: Change to SONGS_LIST to be consistent with the others.

**Comment by SyncedSynapse** [comment]
Is the *Songs* tab on the Artist overview working for you? I don't get any songs there.

**Comment by poisdeux** [comment]
Didn't check well enough. I'll fix this ASAP.

**Comment by poisdeux** [comment]
Should be fixed now.

**Pull request merged**